easybuild/easyconfigs/c/crypt4gh/crypt4gh-1.7-GCC-12.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonBundle'
+
+name = 'crypt4gh'
+version = '1.7'
+
+homepage = 'https://github.com/EGA-archive/crypt4gh'
+description = """crypt4gh is a Python tool to encrypt, decrypt or re-encrypt files, 
+according to the GA4GH encryption file format."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('PyYAML', '6.0'),
+    ('cryptography', '41.0.1'),
+    ('bcrypt', '4.0.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('PyNaCl', '1.5.0', {
+        'modulename': 'nacl',
+        'checksums': ['8ac7448f09ab85811607bdd21ec2464495ac8b7c66d146bf545b0f08fb9220ba'],
+    }),
+    ('docopt', '0.6.2', {
+        'checksums': ['49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491'],
+    }),
```

Comment:
Both of these, in the exact same versions, are provided by `Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb`.